scripts/build.ts

```diff
@@ -108,8 +126,7 @@ const validateProduct = async (zipped: Uint8Array) => {
   const schemaOk = await ajv.validate(manifestSchema, manifestJson);
 
   const definedPaths = [
-    ...manifestJson.content_scripts.flatMap(({ css, js }) => [...css, ...js]),
-    manifestJson.options_page,
+    ...manifestJson.content_scripts.flatMap(({ css, js }) => [...(css || []), ...(js || [])]),
```

Comment:
Question: Why did you change this flatMap implementation? Is this actually required?